src/kernel/include/pgtbl.h

```diff
@@ -47,6 +49,42 @@ typedef enum {
 #define PGTBL_DEPTH 2
 #define PGTBL_ORD 10
 
+/* The page size definitions for composite OS */
+#define PGSZ_128B	7
+#define PGSZ_256B	8
+#define PGSZ_512B	9
+#define PGSZ_1K		10
+#define PGSZ_2K		11
+#define PGSZ_4K		12
+#define PGSZ_8K		13
+#define PGSZ_16K	14
+#define PGSZ_32K	15
+#define PGSZ_64K	16
+#define PGSZ_128K	17
+#define PGSZ_256K	18
+#define PGSZ_512K	19
+#define PGSZ_1M		20
+#define PGSZ_2M		21
+#define PGSZ_4M		22
+
+/* The number of pages in this page table */
+#define PGNUM_2		1
+#define PGNUM_4		2
+#define PGNUM_8		3
+#define PGNUM_16	4
+#define PGNUM_32	5
+#define PGNUM_64	6
+#define PGNUM_128	7
+#define PGNUM_256	8
+#define PGNUM_512	9
+#define PGNUM_1024	10
```

Comment:
I'd prefer naming `PGNO`.
Also, for `>= 1024`, use `1K, 2K, 4K...`. eg. `PGNO_1K`.

Reply:
Ok